README.md

```diff
@@ -338,27 +338,108 @@ the [Applicative](#applicative) and [Plus](#plus) specifications.
 
 ### Foldable
 
-1. `u.reduce` is equivalent to `u.reduce((acc, x) => acc.concat([x]), []).reduce`
+1. `u` is equivalent to `u.foldl((acc, x) => acc.concat([x]), [])`
```

Comment:
Could it be: `u.foldl` is equivalent to `u.foldl((acc, x) => acc.concat([x]), []).foldl`?